src/amo/addonManager.js

```diff
@@ -78,7 +82,7 @@ export function hasAddonManager({
 
 export function hasPermissionPromptsEnabled({
   navigator,
-}: { navigator: PrivilegedNavigatorType } = {}) {
+}: { navigator: PrivilegedNavigatorType } = {}): any | void | boolean {
```

Comment:
ugh? It should only be `boolean | void` (void is for undefined, right?)

Reply:
To be honest, I hadn't checked each and every one of these that the codemod added. I was more concerned with removing the extra code not needed for defaultProps and mapStateToProps, and syncing up the types for those, but this is a good point. I'll look at all the changes to see if they all make sense.